### PR TITLE
Support Version 2.0.0 changes for approve-oam-request

### LINKED ISSUE #133: Support Version 2.0.0 changes for approve-oam-request
1. Generate server stub for the version 2.0.0 specification
2. If the api/openapi.yaml is not updated as per the published version 2.0.0 , then copy the api/openapi.yaml file from the generated stub to the source code
3. No change required in the method signature
4. Remove the current logic that checks the admin profile to validate the oam-requests
5. Fetch the information from the APPData file(administrator-credential-list) to validate the oam-requests

### PR DESCRIPTION
**Pull request by ManasaBM1**
Fixes #133

### CODE REVIEW

**Comment by ManasaBM1** [comment]
Reviewer is @vanithavalluripalli9

File: server/service/AuthorizationApplication.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
+/**
+ * @file This module provides functionality to monitor the whether the registered application are approved within the required time.
```
Comment (vanithavalluripalli9) [inline server/service/AuthorizationApplication.js:2]: please change the comment appropriately

File: server/service/AuthorizationApplication.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+const profile = require('onf-core-model-ap/applicationPattern/onfModel/models/profile/FileProfile');
+const { resolve } = require('path');
+
+const administratorList = 'administrator-credential-list';
+const authorizationValue ='authorization';
+const allowedMethodsValue ='allowed-methods';
+
+
+
```
Comment (vanithavalluripalli9) [inline server/service/AuthorizationApplication.js:19]: Please add short comments for all the methods to understand about the method . Kindly change the file to path ../individualService folder

**Review by vanithavalluripalli9 — CHANGES_REQUESTED** [review]

File: server/service/individualServices/AuthorizationApplication.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
+/**
+ * @file This module provides functionality to monitor the whether the registered application are approved within the required time.
```
Comment (vanithavalluripalli9) [inline server/service/individualServices/AuthorizationApplication.js:2]: Kindly , add the correct use of the module

File: server/service/individualServices/AuthorizationApplication.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+        return
+    }
+} catch(err) {
+    console.error(err)
+  }
+}
+
+/**
+     * @description This function returns the approval status for the provided application .
+     * @returns {promise} string {approvalStatus}
```
Comment (vanithavalluripalli9) [inline server/service/individualServices/AuthorizationApplication.js:40]: kindly , add param related information

File: server/service/individualServices/AuthorizationApplication.js [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+**/
```
Comment (vanithavalluripalli9) [inline server/service/individualServices/AuthorizationApplication.js:41]: same as above

**Review by vanithavalluripalli9 — CHANGES_REQUESTED** [review]

**Review by vanithavalluripalli9 — APPROVED** [review]
Changes looks good . Approved.

**Pull request merged**